### PR TITLE
Small docs fix for Decidable

### PR DESCRIPTION
**Pull request by paf31**
The old law didn't type check.

### CODE REVIEW

File: src/Data/Functor/Contravariant/Divisible.hs [hunk 1]
```diff
@@ -611,8 +611,8 @@ instance Decidable SettableStateVar where
 -- @
 -- 'choose' 'Left' m ('lose' f)  = m
 -- 'choose' 'Right' ('lose' f) m = m
--- 'choose' f ('choose' g m n) o = 'divide' f' m ('divide' 'id' n o) where
---   f' bcd = 'either' ('either' 'id' ('Right' . 'Left') . g) ('Right' . 'Right') . f
+-- 'choose' f ('choose' g m n) o = 'choose' f' m ('choose' 'id' n o) where
+--   f' bcd = 'either' ('either' 'id' 'Left' . g) ('Right' . 'Right') . f
```
Comment (RyanGlScott) [inline src/Data/Functor/Contravariant/Divisible.hs]: Does that `bcd` argument to `f'` belong there? It looks like `f'` is supposed to be written pointfree (and with the way it's written right now, `f'` simply ignores `bcd`).

Comment (paf31) [reply src/Data/Functor/Contravariant/Divisible.hs]: Yes, I think you're right.

Comment (RyanGlScott) [reply src/Data/Functor/Contravariant/Divisible.hs]: OK. Remove `bcd`, and I'll merge :)

Comment (paf31) [reply src/Data/Functor/Contravariant/Divisible.hs]: Updated, sorry for the delay.

Maybe we should fix the line above too (#26)?

**Comment by RyanGlScott** [comment]
Thanks!

File: src/Data/Functor/Contravariant/Divisible.hs (hunk 1 above)
Comment (RyanGlScott) [reply src/Data/Functor/Contravariant/Divisible.hs]: If you have a suggestion for how to fix #26, I'd happily take it as a separate pull request.

**Pull request merged**